src/handlers/opensea-api.ts

```diff
@@ -58,9 +60,34 @@ export const apiGetAccountUniqueTokens = async (
           timeout: 10000, // 10 secs
         })
     );
-    return isPolygon
-      ? parseAccountUniqueTokensPolygon(data)
-      : parseAccountUniqueTokens(data);
+
+    let tokens;
+    if (isPolygon) {
+      // TODO(jxom): migrate this to Async State RFC architecture once it's merged in.
+      const polygonAllowlist = await queryClient.fetchQuery(
```

Comment:
This `fetchQuery` basically means that we want to serve the user with the previously fetched whitelist until it gets stale after 10 minutes (line 78).